tests/test_parallels.py

```diff
@@ -247,8 +252,14 @@ def test_parallel_synchronisation():
     assert(success.status == py_trees.common.Status.SUCCESS)
     print("success_every_second.status == py_trees.common.Status.RUNNING")
     assert(success_every_second.status == py_trees.common.Status.RUNNING)
+    print("success [id: {}] did not get ticked [snapshot: {}]".format(success.id, snapshot_visitor.nodes.keys()))
```

Comment:
Isn't it supposed to get ticked the first time? Missing assert.

Reply:
Right, thanks. Was debugging it and this got left around. Removed.